examples/wordcloud_cn.py

```diff
@@ -0,0 +1,95 @@
+# - * - coding: utf - 8 -*-
+"""
+create wordcloud with chinese
+=======================
+
+Wordcloud is a very good tools, but if you want to create
+Chinese wordcloud only wordcloud is not enough. The file
+shows how to use wordcloud with Chinese. First, you need a
+Chinese word segmentation library jieba, jieba is now the
+most elegant the most popular Chinese word segmentation tool in python.
+You can use 'PIP install jieba'. To install it. As you can see,
+at the same time using wordcloud with jieba very convenient
+"""
+
+import jieba
+jieba.enable_parallel(4)
+# Setting up parallel processes :4 ,but unable to run on Windows
+from os import path
+from scipy.misc import imread
+import matplotlib.pyplot as plt
+# jieba.load_userdict("txt\userdict.txt")
+# add userdict by load_userdict()
+from wordcloud import WordCloud, ImageColorGenerator
+
+d = path.dirname(__file__)
+
+stopwords_path = d + '/wc_cn/stopwords_cn_en.txt'
+# Chinese fonts must be set
+font_path = d + '/fonts/SourceHanSerif/SourceHanSerifK-Light.otf'
```

Comment:
can you include a Chinese font that's not too big with the PR? Otherwise people will have trouble running the example. Or the example could download the font.

Reply:
oh sorry, I just saw it's included, never mind!